### PR TITLE
[enterprise-4.13] bump RHEL versions

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.13

Issue: n/a

Link to docs preview: http://file.rdu.redhat.com/jdohmann/update-about-this-release/release_notes/ocp-4-13-release-notes.html#ocp-4-13-about-this-release

QE review:
- [ ] QE has approved this change.

Additional information: Mark Russell ack'd over slack

### CODE REVIEW

**Review by JoeAldinger — COMMENTED** [review]
small change because of GA in May. I think we also need to change this here as well: [modules/rhel-compute-requirements.adoc]

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -19,7 +19,7 @@ Built on {op-system-base-full} and Kubernetes, {product-title} provides a more s
 {product-title} {product-version} clusters are available at https://console.redhat.com/openshift. With the {cluster-manager-first} application for {product-title}, you can deploy OpenShift clusters to either on-premises or cloud environments.
 
 // Double check OP system versions
-{product-title} {product-version} is supported on {op-system-base-full} 8.4 and 8.5, as well as on {op-system-first} 4.13.
+{product-title} {product-version} is supported on {op-system-base-full} 8.6 and 8.7, as well as on {op-system-first} 4.13.
```
[the remark this answers is not in the document]
Comment (JoeAldinger) [reply release_notes/ocp-4-13-release-notes.adoc:22]: With GA in May the version bump goes to 8.8. Confirmed by @sdodson via slack.

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Pull request merged**